CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 ## release note
+
+## [v2.0.4](https://github.com/skyway/skyway-js-sdk/releases/tag/v2.0.3) - 2019-12-03
```

Comment:
It seems old link(v2.0.3)

Reply:
Thanks! It fixed by https://github.com/skyway/skyway-js-sdk/pull/223/commits/f038b734474adf698bd724ce1b128a59f263c61b .